can't create a custom MergePolicy, because part of the signature is in a private module

**Describe the bug**
Attempt to create a custom MergePolicy, but it requires you to return a MergeCandidate which is private.

**Which version of tantivy are you using?**
0.16.1

**To Reproduce**
Try to create a custom MergePolicy.

~~~~
error[E0603]: module `indexer` is private
   --> src\lib.rs:71:82
    |
71  |     fn compute_merge_candidates(&self, segments: &[SegmentMeta]) -> Vec<tantivy::indexer::merge_policy::MergeCandidate> {
    |                                                                                  ^^^^^^^ private module
    |
~~~~

If your bug is deterministic, can you give a minimal reproducing code?
Some bugs are not deterministic. Can you describe with precision in which context it happened?
If this is possible, can you share your code?

--- a/src/lib.rs
+++ b/src/lib.rs
@@ -237,6 +237,7 @@ pub fn version_string() -> &'static str {
 pub mod merge_policy {
     pub use crate::indexer::DefaultMergePolicy;
     pub use crate::indexer::LogMergePolicy;
+    pub use crate::indexer::MergeCandidate;
     pub use crate::indexer::MergePolicy;
     pub use crate::indexer::NoMergePolicy;
 }

